### ISSUE TITLE
save the soft mask

### ISSUE THREAD

**Issue by Cameltr** [xc-csc101/RAFSwap#4]
Thank you for your excellent work! I want to ask how to save the soft mask? I have made some tries, but it seems different from your result in paper? The following mask image is the test result of using image in your github repository.
![mask](https://user-images.githubusercontent.com/66940842/224545524-adeeb142-3f5c-493e-931b-d968ca8c91be.jpg)

**Comment by johntrimble** [comment]
Yeah, I think something is wrong. Been doing some tests with `inference_celeba.py`:

![1388_1388](https://user-images.githubusercontent.com/156620/230543450-3be88654-22e3-4661-81d3-940e659430d2.jpg)
![1388_14024](https://user-images.githubusercontent.com/156620/230543455-0b71da7e-0da7-41ba-8a9e-c670c7f43991.jpg)
![1388_28068](https://user-images.githubusercontent.com/156620/230543460-cfa38fe0-0d2e-4171-a47d-402db6ef0c3c.jpg)

That's target, source, result, and mask. It looks like there is more than just mask that isn't right. It almost looks like the source is being inverted and that's it.

**Comment by johntrimble** [comment]
Err, wanted to follow up here. My input masks for the source and target were bad. Fixing that didn't make the output mask any better, but at least the result looks more like it should:

![1388_1388](https://user-images.githubusercontent.com/156620/230697048-183c4984-4736-44bb-8ba5-8221ef0785d7.jpg)
![1388_14024](https://user-images.githubusercontent.com/156620/230697052-2af8e61d-4ef5-4784-a394-54607e898891.jpg)
![1388_28068](https://user-images.githubusercontent.com/156620/230697054-19128394-3454-4524-b1d6-7185c4de7439.jpg)
![1388_28132](https://user-images.githubusercontent.com/156620/230697060-d8d816d8-ce48-45e5-831a-fd070366617d.jpg)
